### DISCUSSION TITLE
Admins in github classroom

### DISCUSSION THREAD

**Discussion by Tobeyforce** [community/community#135657] [GitHub Education]
### Hi everyone,

Hi!
I have an organization which I created to manage github classroom related things, as I do some teaching.
I want to invite another person which should be able to create assignments for a specific classroom, and do any teaching related matters related to this particular classroom. Is this achievable currently? The invite page states clearly that Admins or TA's need to be owners of the organization - but making someone else an owner of the organization essentially means they could kick me out or delete things. 

I must say I find roles / permissions quite daunting when using github. Could anyone point me in the direction of what's feasable here? Is it possible to customize a role like this for github classroom? Has anyone done something similar?

![image](https://github.com/user-attachments/assets/95bc397b-ff1d-409a-b30f-886c00d6885a)

**Comment by Irissuu** [comment]
try using [GitHub Teams,](https://github.com/team) you can set specific permissions for each team (such as read, write and admin)
but you can't create custom roles with specific permissions beyond the existing structure
